docs/source/intro/get_started.rst

```diff
@@ -0,0 +1,123 @@
+.. _intro/get_started:
+
+****************
+Getting Started
+****************
+
+Installation
+============
+
+There are numerous routes to setting up the full AiiDA environment, depending on your use case.
+Here we first provide the simplest approach for installation on your local computer.
+
+`Conda <https://docs.conda.io>`_  provides a cross-platform package management system, from which we can install the basic components of the AiiDA infrastructure in an isolated environment:
+
+.. code-block:: console
+
+    $ conda create -n aiida -c conda-forge aiida-core aiida-core.services
+    $ conda activate aiida
+    $ reentry scan
+
+.. note::
+
+    You can also install ``aiida-core`` from `PyPi <https://pypi.org/project/aiida-core>`_
+
+This installs the three core components of AiiDA:
+
+* ``aiida-core``: The main python package and associated CLI ``verdi``.
+* `PostgreSQL <https://www.postgresql.org>`_: A service which manages the database where we store generated data.
+* `RabbitMQ <https://www.rabbitmq.com>`_: A service which manages communication with the processes that we run.
+
+To initialise a database cluster with PostgreSQL and start the service:
+
+.. code-block:: console
+
+    $ initdb -D mylocal_db
+    $ pg_ctl -D mylocal_db -l logfile start
```

Comment:
Could this be rolled into `conda activate aiida`?

Reply:
see above comment